frontend/src/metabase/query_builder/components/DataSelector.jsx

```diff
@@ -243,12 +243,16 @@ export default class DataSelector extends Component {
 
     renderTablePicker() {
         const schema = this.state.selectedSchema;
+
         const hasMultipleDatabases = this.props.databases.length > 1;
+        const hasMultipleSchemas = schema && schema.database && _.chain(schema.database.tables).pluck("schema").uniq().value().length > 1;
```

Comment:
what's ` _.chain(schema.database.tables).pluck("schema").uniq().value().length`  all about?

seems a little too much of a one-liner.